media/js/firefox/new/scene1-fx-experiment.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    var client = Mozilla.Client;
+    if(client.isFirefox){
+        client.getFirefoxDetails(function(details) {
+            // only initialize experiment if current Firefox user not logged into FxA
+            if (details.firefox && !details.setup && (client.FirefoxVersion.indexOf(62) === 0)) {
```

Comment:
The call to `getfirefoxDetails()` here doesn't return the property `details.firefox`, so the experiment here is not currently executing. 

P.S - Checking for `client.FirefoxVersion.indexOf(62) === 0` also made this a bit difficult to test / debug 😕

Reply:
Should be calling `getFxaDetails` 😖

Reply:
✅